src/adagrams.js

```diff
@@ -1,15 +1,121 @@
+const LETTER_POOL = {
+  A: 9,
+  B: 2,
+  C: 2,
+  D: 4,
+  E: 12,
+  F: 2,
+  G: 3,
+  H: 2,
+  I: 9,
+  J: 1,
+  K: 1,
+  L: 4,
+  M: 2,
+  N: 6,
+  O: 8,
+  P: 2,
+  Q: 1,
+  R: 6,
+  S: 4,
+  T: 6,
+  U: 4,
+  V: 2,
+  W: 2,
+  X: 1,
+  Y: 2,
+  Z: 1,
+};
 export const drawLetters = () => {
-  // Implement this method for wave 1
+
+  let drawedLetters = [];
+  const numAllowedinHand = 10;
+  let allLetters = [];
+
+  for (const [key, value] of Object.entries(LETTER_POOL)){
+    for (let i = 0; i < value; i++){
+      allLetters.push(key);
+    };
+  };
```

Comment:
Nicely done. I'd prefer this logic to be in a helper function like `createLetterPool` so that `drawLetters` is more single responsibility and doesn't need to create a pool __and__ draw the letters. 
```suggestion
  let drawedLetters = [];
  const numAllowedinHand = 10;
  let allLetters = createLetterPool();

  // rest of your logic ...
```